channeldb/payments.go

```diff
@@ -421,6 +421,154 @@ func fetchPayment(bucket *bbolt.Bucket) (*MPPayment, error) {
 	return p.ToMPPayment(), nil
 }
 
+// PaymentsQuery represents a query to the payments database starting or ending
+// at a certain offset index. The number of retrieved records can be limited.
+type PaymentsQuery struct {
+	// IndexOffset determines the starting point of the payments query and
+	// is always exclusive. In normal order, the query starts one index
+	// higher than the IndexOffset, in reversed order, the query ends one
+	// index lower than the IndexOffset.
+	IndexOffset uint64
+
+	// NumMaxPayments is the maximal number of payments returned in the
+	// payments query.
+	NumMaxPayments uint64
+
+	// Reversed gives a meaning to the IndexOffset. If reversed is set to
+	// true, the query will fetch payments with indices lower than the
+	// IndexOffset.
+	Reversed bool
+
+	// If IncludeIncomplete is true, then return payments that have
+	// not yet fully completed. This means that pending payments,
+	// as well as failed payments will show up if this
```

Comment:
nit, line wrap looks odd here + no capital for `True` in comment.

Reply:
Improved it.